### PR TITLE
OCPBUGS-6875: Clarified the Compliance Operator supported profiles table

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.9+

Issue:
https://issues.redhat.com/browse/OCPBUGS-6875

Link to docs preview (VPN required) :
[Compliance profiles](http://file.rdu.redhat.com/antaylor/OCPBUGS-6875/security/compliance_operator/compliance-operator-supported-profiles.html)
[Compliance Operator profile types](http://file.rdu.redhat.com/antaylor/OCPBUGS-6875/security/compliance_operator/compliance-operator-understanding.html#compliance_profile_typesunderstanding-compliance)

QE review:
- [x] QE has approved this change.

Additional information:
This pull request attempts to clarify how to use the Node- and Platform-specific compliance profiles, while also cleaning up footnote formatting. Currently, footnote [1] is below Additional resources in [published content](https://docs.openshift.com/container-platform/4.12/security/compliance_operator/compliance-operator-supported-profiles.html#additional-resources-compliance-operator-).

### CODE REVIEW

File: modules/compliance-supported-profiles.adoc [hunk 1]
```diff
@@ -8,108 +8,127 @@
...
+|Platform
 |0.1.52+
 |link:https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/800-53[NIST SP-800-53 Release Search]
 |`x86_64`
-|===
\ No newline at end of file
+|===
+[.small]
+1. To locate the CIS {product-title} v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on *Kubernetes* and then *Download Latest CIS Benchmark*, where you can then register to download the benchmark.
+2. Node profiles must be used with the relevant Platform profile. For example, `ocp4-high-node` Node profiles must be used in `ocp4-high` {product-title} environments.
```
Comment (rhmdnd) [inline modules/compliance-supported-profiles.adoc:134]: Do we want to use this opportunity to describe the difference between a Node and Platform type? Or should we add that in a separate part of the documentation and link to it? Perhaps the "understand the complinace operator" section?

Comment (sheriff-rh) [reply modules/compliance-supported-profiles.adoc:134]: I like the idea of using the "Understanding" page, and then we can cross reference to it. I would be happy to include those changes to this PR.

Comment (sheriff-rh) [reply modules/compliance-supported-profiles.adoc]: I updated the PR to include the above changes discussed: [Compliance Operator profile types](http://file.rdu.redhat.com/antaylor/OCPBUGS-6875/security/compliance_operator/compliance-operator-understanding.html#compliance_profile_typesunderstanding-compliance)

File: modules/compliance-supported-profiles.adoc [hunk 2]
```diff
@@ -8,108 +8,127 @@
...
 |ocp4-moderate
 |NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift - Platform level
+|Platform
 |0.1.39+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
 |`x86_64`
 
 |rhcos4-e8
 |Australian Cyber Security Centre (ACSC) Essential Eight
+|Platform
```
Comment (xiaojiey) [inline modules/compliance-supported-profiles.adoc:55]: All rhcos4 related rules should be Node Type.
@jhrozek Correct me if I am wrong.

File: modules/compliance-supported-profiles.adoc [hunk 3]
```diff
@@ -8,108 +8,127 @@
...
 |0.1.39+
 |link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
 |`x86_64`
 
 |rhcos4-moderate
```
Comment (xiaojiey) [inline modules/compliance-supported-profiles.adoc:60]: The same as above.

File: modules/compliance-supported-profiles.adoc [hunk 4]
```diff
@@ -8,108 +8,127 @@
...
 |NIST 800-53 Moderate-Impact Baseline for Red Hat Enterprise Linux CoreOS
+|Platform
```
Comment (xiaojiey) [inline modules/compliance-supported-profiles.adoc:62]: The same as above

File: modules/compliance-supported-profiles.adoc [hunk 5]
```diff
@@ -8,108 +8,127 @@
...
 |ocp4-nerc-cip-node
 |North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for the Red Hat OpenShift Container Platform - Node level
+|Node ^[2]^
 |0.1.44+
 |link:https://www.nerc.com/pa/Stand/Pages/CIPStandards.aspx[NERC CIP Standards]
 |`x86_64`
 
 |rhcos4-nerc-cip
 |North American Electric Reliability Corporation (NERC) Critical Infrastructure Protection (CIP) cybersecurity standards profile for Red Hat Enterprise Linux CoreOS
+|Platform
```
Comment (xiaojiey) [inline modules/compliance-supported-profiles.adoc:90]: The same as above

File: modules/compliance-supported-profiles.adoc [hunk 6]
```diff
@@ -8,108 +8,127 @@
...
 |ocp4-high-node
 |NIST 800-53 High-Impact Baseline for Red Hat OpenShift - Node level
+|Node ^[2]^
 |0.1.52+
 |link:https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/800-53[NIST SP-800-53 Release Search]
 |`x86_64`
 
 |rhcos4-high
 |NIST 800-53 High-Impact Baseline for Red Hat Enterprise Linux CoreOS
+|Platform
```
Comment (xiaojiey) [inline modules/compliance-supported-profiles.adoc:127]: The same as above.

File: modules/compliance-supported-profiles.adoc (hunk 2 above)
Comment (sheriff-rh) [reply modules/compliance-supported-profiles.adoc:55]: Good catch - I believe that is correct as well, based on upstream documentation. I'll wait for Jakub to verify as well.

Comment (jhrozek) [reply modules/compliance-supported-profiles.adoc:55]: @xiaojiey is right, all the RHCOS profiles are Node type.

**Review by rhmdnd — COMMENTED** [review]
/lgtm

**Review by prb112 — APPROVED** [review]
LGTM

File: modules/compliance-supported-profiles.adoc [hunk 7]
```diff
@@ -8,108 +8,127 @@
 The Compliance Operator provides the following compliance profiles:
 
 .Supported compliance profiles
-[cols="10%,40%,10%,40%,10%", options="header"]
+[cols="10%,40%,10%,10%,40%,10%", options="header"]
```
Comment (nalhadef) [inline modules/compliance-supported-profiles.adoc:11]: Up around line 6: Add concept_type

**Comment by nalhadef** [comment]
@sheriff-rh Just one comment......

File: modules/compliance-supported-profiles.adoc (hunk 7 above)
Comment (sheriff-rh) [reply modules/compliance-supported-profiles.adoc:11]: Good catch!

**Pull request merged**